--- origsrc/kdeplasma-addons-4.4.3/applets/kdeobservatory/cmake/modules/FindQwt.cmake	2009-11-30 18:35:54.000000000 -0600
+++ src/kdeplasma-addons-4.4.3/applets/kdeobservatory/cmake/modules/FindQwt.cmake	2010-05-05 16:37:03.712903600 -0500
@@ -37,7 +37,7 @@ IF( QT4_FOUND )
 		STRING(REGEX REPLACE ".*#define[\\t\\ ]+QWT_VERSION_STR[\\t\\ ]+\"([0-9]+\\.[0-9]+\\.[0-9]+)\".*" "\\1" Qwt_VERSION "${QWT_GLOBAL_H}")
 
 		# Find Qwt5 library linked to Qt4
-		FIND_LIBRARY( Qwt5-Qt4_TENTATIVE_LIBRARY NAMES qwt5-qt4 qwt-qt4 qwt5 qwt )
+		FIND_LIBRARY( Qwt5-Qt4_TENTATIVE_LIBRARY NAMES qwt5-qt4 qwt-qt4 qwt5 qwt PATHS ${QT_LIBRARY_DIR} )
 		IF( UNIX AND NOT CYGWIN)
 			IF( Qwt5-Qt4_TENTATIVE_LIBRARY )
 				#MESSAGE("Qwt5-Qt4_TENTATIVE_LIBRARY = ${Qwt5-Qt4_TENTATIVE_LIBRARY}")
